packages/itinerary-body/src/TransitLegBody/index.tsx

```diff
@@ -287,7 +292,10 @@ class TransitLegBody extends Component<Props, State> {
                 )}
               </S.TransitLegDetailsHeader>
               {/* IntermediateStops expanded body */}
-              <AnimateHeight duration={500} height={stopsExpanded ? "auto" : 0}>
+              <AnimateHeight
+                duration={shouldAnimate ? 500 : 0}
```

Comment:
since you're doing this check multiple times, you could assign the value to variable outside the mark-up and then re-use as a prop for the instances of `AnimateHeight`, something like:

```js
const animationDuration = shouldAnimate ? 500 : 0;

<AnimateHeight duration={animationDuration} />
```